lib/Widget/MastodonProvider.php

```diff
@@ -157,4 +170,45 @@ public function getDataModifiedDt(DataProviderInterface $dataProvider): ?Carbon
     {
         return null;
     }
+
+    /**
+     * Get Mastodon Account Id from username
+     * @throws GuzzleException
+     */
+    private function getAccountId(mixed $uri, DataProviderInterface $dataProvider)
+    {
+        $uri = rtrim($uri, '/').'/api/v1/accounts/lookup?';
+
+        $httpOptions = [
+            'timeout' => 20, // wait no more than 20 seconds
+            'query' => [
+                'acct' => $dataProvider->getProperty('userName')
+            ],
+        ];
+        $response = $dataProvider
+            ->getGuzzleClient($httpOptions)
+            ->get($uri);
+
+        $result = json_decode($response->getBody()->getContents(), true);
+
+        $this->getLog()->debug('Mastodon: getAccountId: ID ' . $result['id']);
+
+        return $result['id'];
+    }
+
+    /**
+     * Get Mastodon account public status
+     * @throws GuzzleException
+     */
+    private function getResult(mixed $uri, $accountId, $httpOptions)
```

Comment:
Is this really `getResult` ?

It seems like this is only used for account search?

Is $uri really mixed?